### PR TITLE
Explain meaning of issue & pull request labels

### PR DESCRIPTION
**Pull request by Ichimonji10**
Define the meaning of the following labels:

* Issue Type: Bug
* Issue Type: Discussion
* Issue Type: Plan
* Issue Type: Test Case
* Status: LGTM
* Status: Needs Review
* Status: Needs Work

Add this explanation to the documentation at Pulp Smash → About →
Contributing → Labels.

### CODE REVIEW

**Comment by Ichimonji10** [comment]
Hunh. I'm not even sure how I removed the "in progress" label. I think we might've been touching the issue concurrently. Anyway. This proposal is totally up for discussion. Please comment.

**Comment by Ichimonji10** [comment]
Of interest: http://www.ianbicking.org/blog/2014/03/use-github-issues-to-organize-a-project.html

File: docs/about.rst [hunk 1]
```diff
@@ -117,6 +117,53 @@ on.
...
+~~~~~~
+
+`Labels`_ are used to categorize issues and pull requests. The "Issue Type"
+labels may be applied to issues, and the "Status" labels may be applied to
+either issues or pull requests.
+
+============  ====================  ===============================
+Label Prefix  Applicable to Issues  Applicable to Pull Requests
+============  ====================  ===============================
+Issue Type:   ✓                     ✗
```
Comment (elyezer) [inline docs/about.rst]: :+1:

**Comment by elyezer** [comment]
Brief and direct to the point, really good addition. I think when this PR is merged we are good to remove other labels and stick with the ones described here.

**Comment by danuzclaudes** [comment]
The classification of Bug/Discussion/Plan/Test Case looks very clear to me! I'll apply them to my current and future issues :  D

**Comment by Ichimonji10** [comment]
Great! Given that the proposal went over well during our meeting, and given that the two main devs are happy with the implementation, I'll merge.

**Pull request merged**